README.md

```diff
@@ -112,3 +113,29 @@ unserializable string.
 
 Exceptions will also be serialized. Stack Traces, filenames, line numbers and previous exceptions will be omitted
 when `kernel.debug` is set to `false` (= in productive environments) to avoid leaking potentially sensitive information.
+
+### Serialized Protobuf view
+If you wish to return Protobuf.
```

Comment:
As I understand the code, the mapper should return a Protobuf object. Is that correct? You should then state it here in the readme to be better understandable